inventory.yml

```diff
@@ -0,0 +1,42 @@
+all:
+  vars:
+
+    # SRO namespace. Do not change
+    sro_namespace: openshift-special-resource-operator
+
+    # Path to pull secret for local registry and standard registries
+    pull_secret: /tmp/pull-secret.txt
+    
+    # Path to kubeconfig for the ACM hub cluster
+    kubeconfig: /tmp/kubeconfig
+    
+    # Path to repo git pull directory
+    repopath: /tmp/openshift-acm-oot-drv
+
+    # Local registry domain name and port.
+    registry: "cnfde2.ptp.lab.eng.bos.redhat.com:5000"
```

Comment:
Add details to the readme to show what variables need to change/update.  for example, the local registry domain name and port should be updated.

Reply:
Will do